### DISCUSSION TITLE
Filters doesn't work properly

### DISCUSSION THREAD

**Discussion by anardil** [bagisto/bagisto#5950] [General]
Hello,

I've got a question about attributes displayed on Layered navigation.
Example: I've got 2 products in the category Tshirt:
- 1 Tshirt color red with size S
- 1 Tshirt color blue with size XL

My question is why when I make a filter with the color "red" I can see the sizes S and XL? 
Normally it should have only the size S.

<img width="501" alt="Capture d’écran 2022-03-04 à 09 38 59" src="https://user-images.githubusercontent.com/471437/156730537-6dc8e724-3723-4fcc-b6c9-9ac0240117c3.png">

How can I have only the size S show in filter attribute size?

Thanks for your help.

**Comment by amit-webkul** [comment]
Hello @anardil ,
I got your point and also understand where you are facing the issue.

If you need to add filters after adding form attributes, you have the option to choose from the category edit page which attribute filters you want to make available on the front end of the site. 

I have added a screenshot link where you can find the section to enable attributes for the front end. [Link](https://prnt.sc/UtQK8cO21Y7h)

After following these steps, I hope your error has been resolved. If it has not been resolved, please feel free to provide your response here. 

**Note:- The same category should be added in the variant product.**

**Comment by Amitk-webkul** [comment]
Hi @anardil, I believe you've found your solution. you've found your solution. If you have any more questions in the future, don't hesitate to reach out. Just go ahead and close this discussion for now.
